uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/RouteContextSupportTest.java

```diff
@@ -0,0 +1,125 @@
+package org.jboss.tools.fuse.ui.bot.tests;
+
+import static org.jboss.tools.fuse.reddeer.wizard.NewFuseIntegrationProjectWizardDeploymentType.STANDALONE;
+import static org.jboss.tools.fuse.reddeer.wizard.NewFuseIntegrationProjectWizardRuntimeType.KARAF;
+import static org.junit.Assert.assertTrue;
+import java.util.Collection;
+
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitUntil;
+import org.eclipse.reddeer.eclipse.ui.console.ConsoleView;
+import org.eclipse.reddeer.eclipse.ui.views.contentoutline.ContentOutline;
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.requirements.cleanworkspace.CleanWorkspaceRequirement;
+import org.jboss.tools.fuse.reddeer.ProjectTemplate;
+import org.jboss.tools.fuse.reddeer.editor.CamelEditor;
+import org.jboss.tools.fuse.reddeer.projectexplorer.CamelProject;
+import org.jboss.tools.fuse.reddeer.utils.LogChecker;
+import org.jboss.tools.fuse.reddeer.utils.ProjectFactory;
+import org.jboss.tools.fuse.reddeer.wizard.CamelXmlFileWizard;
+import org.jboss.tools.fuse.ui.bot.tests.utils.EditorManipulator;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.eclipse.reddeer.swt.api.TreeItem;
+import org.eclipse.reddeer.swt.condition.ShellIsAvailable;
+
+/**
+ * @author fpospisi
+ */
+@RunWith(RedDeerSuite.class)
+public class RouteContextSupportTest {
+	public static final String PROJECT_NAME = "routeContextSupportTest";
+	public static final String ROUTES_NAME = "camel-routes.xml";
+	
+	/**
+	 * <p>
+	 * routeContext support test
+	 * </p>
+	 * <b>Steps</b>
+	 * <ol>
+	 * <li>Create new project: STANDALONE - KARAF - CBR_SPRING.</li>
+	 * <li>In source tab change XML to external <routeContext> "camel-routes.xml" reference.</li>
+	 * <li>Create new XML - camel-routes.xml.</li>
+	 * <li>Edit 'Source' in camel-routes.xml.</li>
+	 * <li>Check camel-routes.xml 'Design' tab for containing items setted up in source. </li>
+	 * <li>Run as Local Camel Context (without tests) and check for Fuse related errors.</li>
+	 * </ol>
+	 */
+	
+	@Before
```

Comment:
maybe you can consider usage of `@BeforeClass` instead of `@Before` WDYT?

see https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall for differences